### PR TITLE
Initial submission

### PR DESCRIPTION
**Pull request by csjall**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-system-monitor-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?
This is the initial submission of the system monitor custom device. This is a refresh of the existing [system monitor custom device](https://github.com/NIVeriStandAdd-Ons/System-Monitor-Custom-Device) which uses the inline async api to publish channels.

### Why should this Pull Request be merged?

The system monitor custom device is helpful for monitoring CPU and memory usage.

### What testing has been done?

image

### CODE REVIEW

**Comment by csjall** [comment]
> The [automated build](http://coordinator/job/VeriStand/job/ni/job/niveristand-system-monitor-custom-device/view/change-requests/job/PR-1/1/) failed on `Source\Help\Build CHM.vi`.

The build machine does not have the [VeriStand Development Tools](https://forums.ni.com/t5/NI-VeriStand-Add-Ons-Documents/VeriStand-Development-Tools-VIPC/ta-p/3632685) installed. Even after installing the VeriStand Development Tools I still get build failure due to dependency on 2015 ProjectTemplate.

http://coordinator/job/VeriStand/job/ni/job/niveristand-system-monitor-custom-device/view/change-requests/job/PR-1/

**Comment by rtzoeller** [comment]
Can you test on both Linux PXI and CompactRIO chassis?

There is a PXI chassis available at `10.2.64.15`, @Karl-G1 has a CompactRIO combo chassis which you can ask about.

**Comment by csjall** [comment]
image

**Comment by buckd** [comment]
Your .lvlib files should define an icon template that's not just a color. It should indicate it's a system monitor and whether it's engine or system explorer. I won't hold up the merge for that though.

File: control [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+XB-Plugin: file
+Description: Provides support for the system monitor custom device for NI VeriStand {veristand_version}.
+XB-Eula: eula-ni-standard
+Priority: standard
+Homepage: http://www.ni.com
+XB-LanguageSupport: en
+XB-UserVisible: yes
+Section: Add-Ons
+XB-DisplayVersion: {display_version}
+XB-DisplayName: System Monitor for VeriStand {veristand_version}
```
Comment (buckd) [inline control]: By convention, the control file should have a blank line at the end of the file.

File: .gitignore [hunk 2]
```diff
@@ -15,5 +15,5 @@
 *.aliases
 *.lvlps
 
-# Built directory
-/Built
+# Output directory
+Built/
```
Comment (buckd) [inline .gitignore]: By convention, .gitignore should have a blank line at the end of the file.

**Pull request merged**